cognite/client/data_classes/workflows.py

```diff
@@ -1177,21 +1177,76 @@ def dump(self, camel_case: bool = True, as_external_id: bool = False) -> list[di
         return [workflow_id.dump(camel_case, as_external_id_key=as_external_id) for workflow_id in self.data]
 
 
-WorkflowTriggerType: TypeAlias = Literal["scheduledTrigger"]
+WorkflowTriggerType: TypeAlias = Literal["schedule"]
 
 
 class WorkflowScheduledTriggerRule(CogniteObject):
     """
     This class represents a scheduled trigger rule.
 
     Args:
-        cron_spec (str | None): The cron specification for the scheduled trigger.
+        cron_expression(str | None): The cron specification for the scheduled trigger.
     """
 
-    def __init__(self, cron_spec: str | None = None) -> None:
-        self.type: WorkflowTriggerType = "scheduledTrigger"
-        # self.trigger_type: WorkflowTriggerType = "scheduledTrigger"
-        self.cron_spec = cron_spec
+    def __init__(self, cron_expression: str | None = None) -> None:
+        self.trigger_type: WorkflowTriggerType = "schedule"
+        self.cron_expression = cron_expression
+
+
+class WorkflowTriggerCreate(CogniteResource):
+    """
+    This class represents a workflow trigger for creation.
+
+    Args:
+        external_id (str): The external ID provided by the client. Must be unique for the resource type.
+        trigger_rule (WorkflowScheduledTriggerRule): The trigger rule of the workflow version trigger.
+        workflow_external_id (str): The external ID of the workflow.
+        workflow_version (str): The version of the workflow.
+        input (dict | None): The input data of the workflow version trigger. Defaults to None.
+        authentication (dict | None): The authentication of the workflow version trigger. Defaults to None.
```

Comment:
The descriptions are not very useful. For example, for input, we could say "this object will be the workflow input to any execution started by the trigger".